### PR TITLE
Feature/update cassandra

### PR DESCRIPTION
**Pull request by 4c74356b41**

### CODE REVIEW

File: tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-"keystorePass":"mypassword",
-"keyAlias":"clientkeyalias",
-"credentialsList":[
-    {"host":"localhost","port":-1,"user":"user","password":"password"}
+    "maxConnectionsTotal": 100,
+    "maxConnectionsPerRoute": 10,
+    "truststoreType": "JKS",
+    "truststoreProvider": "SUN",
+    "truststoreFile": "file:/opt/tip-wlan/certs/truststore.jks",
+    "truststorePass": "mypassword",
```
Comment (eugenetaranov-opsfleet) [inline tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json:7]: should not we parametrize the password here?

File: tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+    "keystoreType": "JKS",
+    "keystoreProvider": "SUN",
+    "keystoreFile": "file:/opt/tip-wlan/certs/client_keystore.jks",
+    "keystorePass": "mypassword",
```
Comment (eugenetaranov-opsfleet) [inline tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json:11]: should not we parametrize the password here?

File: tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
+    "keyAlias": "clientkeyalias",
+    "credentialsList": [
+        {
+            "host": "localhost",
+            "port": -1,
+            "user": "user",
+            "password": "password"
```
Comment (eugenetaranov-opsfleet) [inline tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json:18]: should not we parametrize the password here?

File: tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json (hunk 1 above)
Comment (eugenetaranov-opsfleet) [reply tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json:7]: nvm, just realized the formatting was changed.. needs to be fixed in a separate PR

File: tip-wlan/templates/cassandra-dependencies.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Namespace }}-common-cassandra-certs
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  truststore: {{ .Files.Get "resources/certs/truststore.jks" | b64enc }}
+  truststore-password: {{ "mypassword" | b64enc }}
```
Comment (eugenetaranov-opsfleet) [inline tip-wlan/templates/cassandra-dependencies.yaml:10]: probably mypassword needs to be taken from values

File: tip-wlan/templates/cassandra-dependencies.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  keystore: {{ .Files.Get "resources/certs/cassandra_server_keystore.jks" | b64enc }}
+  keystore-password: {{ "mypassword" | b64enc }}
```
Comment (eugenetaranov-opsfleet) [inline tip-wlan/templates/cassandra-dependencies.yaml:12]: same here

**Comment by 4c74356b41** [comment]
https://github.com/Telecominfraproject/wlan-cloud-helm/pull/49

**Pull request closed without merging**